Cargo.toml

```diff
@@ -11,15 +11,15 @@ name = "tantivy"
 crate-type = ["cdylib"]
 
 [build-dependencies]
-pyo3-build-config = "0.16.3"
+pyo3-build-config = { version = "0.18.0" }
 
 [dependencies]
-chrono = "0.4.19"
-tantivy = "0.17"
-itertools = "0.10.3"
-futures = "0.3.21"
-serde_json = "1.0.64"
+chrono = { version = "0.4.23" }
+tantivy = { version = "0.19.2" }
+itertools = { version = "0.10.5" }
+futures = { version = "0.3.26" }
+serde_json = { version = "1.0.91" }
```

Comment:
I dont think this change is worth it (using version = xxx) we don't use any of the features so we can just use the standard `dep = "x.x.x"` format.

Reply:
Done